lab5.py

```diff
@@ -0,0 +1,61 @@
+class File:
+    # pylint: disable=R0903
+    def __init__(self, name, extension, size):
+        self.name = name
+        self.extension = extension
+        self.size = size
+
+    def __str__(self):
+        return f"{self.name}.{self.extension} ({self.size} bytes)"
+
+
+class Folder:
+    def __init__(self, name):
+        self.name = name
+        self.files = []
+        self.folders = []
+
+    def add_file(self, file):
+        self.files.append(file)
+
+    def add_folder(self, folder):
+        self.folders.append(folder)
+
+    def print_contents(self, indent=0):
+        print("  " * indent + self.name + "/")
+        for file in self.files:
+            print("  " * (indent + 1) + str(file))
+        for folder in self.folders:
+            folder.print_contents(indent + 1)
+
+    def get_longest_path(self):
+        if not self.folders:
+            return [self.name]
+
+        longest_path = max(folder.get_longest_path() for folder in self.folders)
+        return [self.name] + longest_path
+
+
+def main():
+    file1 = File("file1", "txt", 1024)
+    file2 = File("file2", "doc", 2048)
+    file3 = File("file3", "jpg", 3072)
```

Comment:
its an instance of class "file" and its name can be defined as you wish. i dont think renaming them is gonna bring any big difference, because we can identify files by their extension.